src/main/resources/com/cloudbees/plugins/credentials/impl/CertificateCredentialsImpl/UploadedKeyStoreSource/configUpload.js

```diff
@@ -0,0 +1,57 @@
+// multiple objects named "password" in the form =>
+// extend findNearBy to allow selecting by id
+if (!findNearBy.patched) {
```

Comment:
I think allowing IDs or CSS selectors in `checkDependsOn` instead of just names would help with ambiguities. Should be probably done in core, but for now can be patched in plugins like this.

@daniel-beck do you have a better idea?

Reply:
@jenkinsci/core-security-review Any objection to this technique?